scripts/core/sampler.yml

```diff
@@ -25,16 +25,16 @@ desc: "Sampler addressing mode"
 class: $xSampler
 name: $x_sampler_addressing_mode_t
 etors:
-    - name: MIRRORED_REPEAT
-      desc: "Mirrored Repeat"
-    - name: REPEAT
-      desc: "Repeat"
-    - name: CLAMP
-      desc: "Clamp"
-    - name: CLAMP_TO_EDGE
-      desc: "Clamp to edge"
     - name: NONE
       desc: "None"
+    - name: CLAMP_TO_EDGE
+      desc: "Clamp to edge"
+    - name: CLAMP
+      desc: "Clamp"
+    - name: REPEAT
+      desc: "Repeat"
+    - name: MIRRORED_REPEAT
+      desc: "Mirrored Repeat"
```

Comment:
If the values of these enums matter, we should be explicit about it by adding `value: "<value>"` to each of them.